lib/cogctl/actions/groups.ex

```diff
@@ -7,43 +7,35 @@ defmodule Cogctl.Actions.Groups do
   end
 
   def run(_options, _args, _config, endpoint),
-    do: with_authentication(endpoint, &do_list/1)
+  do: with_authentication(endpoint, &do_list/1)
 
   defp do_list(endpoint) do
     case CogApi.HTTP.Groups.index(endpoint) do
       {:ok, groups} ->
-        group_attrs = for group <- groups do
-          [group.name, group.id]
-        end
-
-        display_output(Table.format([["NAME", "ID"]] ++ group_attrs, true))
-      {:error, error} ->
-        display_error(error["errors"])
+        group_rows = Enum.map(groups, fn(group) -> [group.name, group.id] end)
+        display_output(Table.format([["NAME", "ID"]] ++ group_rows, true))
+      {:error, message} ->
+        display_error(message)
     end
   end
 
-  def render_memberships(%{"members" => %{"users" => users, "groups" => groups, "roles" => roles}}) do
-    user_attrs = for user <- users do
-      [user["username"], user["id"]]
-    end
-
-    group_attrs = for group <- groups do
-      [group["name"], group["id"]]
-    end
-
-    role_attrs = for role <- roles do
-      [role["name"], role["id"]]
-    end
+  def render(group) do
+    format_table(group) |> display_output
+  end
 
-    """
-    User Memberships
-    #{Table.format([["USERNAME", "ID"]] ++ user_attrs, true)}
+  def render(group, message) do
+    message <> "\n\n" <> format_table(group) |> display_output
```

Comment:
I like the `format_table` function. I think `format_users` and `format_roles` would be nice if we are including the IDs and keep the list looking readable. That may be better to do for the refactor, though.